src/fix_move.cpp

```diff
@@ -1354,6 +1371,11 @@ void FixMove::set_arrays(int i)
         if (quat) {
           // qoriginal = f(quat,-delta);   // NOTE: edit this line
```

Comment:
Any insight on the original note to edit this line ?

Reply:
Good point, I meant to check on this. I have no insight whatsoever, maybe @sjplimp would know? Looks like this line goes back quite a few years.